src/core/backbone_router/bbr_local.cpp

```diff
@@ -252,8 +252,7 @@ void Local::HandleBackboneRouterPrimaryUpdate(Leader::State aState, const Backbo
 
         if (!Get<Mle::MleRouter>().IsLeader())
         {
-            delay += Random::NonCrypto::GetUint8InRange(0, mRegistrationJitter < 255 ? mRegistrationJitter + 1
-                                                                                     : mRegistrationJitter);
+            delay += Random::NonCrypto::GetUint8InRange(0, Min(mRegistrationJitter + 1, 255));
```

Comment:
If `mRegistrationJitter ` equals to 300, the result of the original code is 300. The result of the new code is 255.